routes/socket/user-requests.js

```diff
@@ -354,6 +354,7 @@ module.exports.sendGameInfo = (socket, uid) => {
 			if (player) {
 				player.leftGame = false;
 				player.connected = true;
+				if (game.general) game.general.allPlayersLeft = null;
```

Comment:
The field name of `allPlayersLeft` is slightly unintuitive and doesn't match with `timeCreated` and `timeCompleted` -- I'd recommend `timeAbandoned` as a simple name but anything works as long as it's clear it refers to a time and not a condition.